merlin/schema/schema.py

```diff
@@ -223,9 +370,42 @@ def without(self, col_names):
             ]
         )
 
-    def get(self, col_name, default=None):
+    def get(self, col_name: str, default: ColumnSchema = None) -> ColumnSchema:
+        """Get a ColumnSchema by name
+
+        Parameters
+        ----------
+        col_name : str
+            Name of the column to get
+        default: ColumnSchema :
+            Default value to return if column is not found.
+             (Default value = None)
+
+        Returns
+        -------
+        ColumnSchema
+            Retrieved column schema (or default value, if not found)
+
+        """
         return self.column_schemas.get(col_name, default)
 
+    @property
+    def first(self) -> ColumnSchema:
+        """Returns the first ColumnSchema in the Schema. Useful for cases where you select down
+        to a single column via select_by_name or select_by_tag, and just want the value
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+
```

Comment:
I think we can remove Parameters here - but we probably should include the Returns type (rather than leaving blank)